tools/lint.js

```diff
@@ -117,13 +117,6 @@ async function clippy() {
       "--",
       "-D",
       "warnings",
-      "-A",
-      // https://github.com/rust-lang/rust-clippy/issues/407
-      "clippy::extra_unused_lifetimes",
-      "-A",
-      // https://github.com/rust-lang/rust-clippy/issues/7271
-      // False positives in core/resources.rs for lifetime elision.
-      "clippy::needless_lifetimes",
```

Comment:
In the future, these should probably go in .cargo/config.toml so it also works with `cargo clippy`.